controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -24,16 +24,22 @@ def recording_path() -> Path:
     date = now.date().isoformat()
     # Windows doesn't like colons in filenames
     name = now.isoformat().replace(':', '-')
-    return Path(date) / name / '{}.html'.format(name)
+    return Path(date) / name / '{}'.format(name)
```

Comment:
Perhaps lose the `.format` entirely?

Aside: this is likely to conflict with my changes in #180, but we'll just have to deal with that.